rskj-core/src/main/resources/config/regtest.conf

```diff
@@ -24,12 +24,14 @@ peer {
         # the peer window will show
         # only what retrieved by active
         # peer [true/false]
-        enabled = false
+        enabled = true
 
         # List of the peers to start
         # the search of the online peers
         # values: [ip:port]
         ip.list = [ ]
+
+        allowMultipleConnectionsPerHost = true
```

Comment:
`regtest.conf` is only one network config among others, like `mainnet`, `testnet` etc. (and I believe PeerExplorer is disabled in `regtest`). Check out `reference.conf` file, which is a base config for all networks.